index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div>
+            <a href="#" class="logo">My Logo</a>
+        </div>
+        <nav class="menu">
+            <img src="./assets/icons/Union.png" alt="Menu Icon">
+        </nav>
+    </header>
+
+    <main>
+        <section class="headline-section">
+            <h1 class="heading-1">I'm Abdul Glad to see you!</h1>
```

Comment:
- [Optional] Please use a `<br>` tag to make the `Glad to see you!` , go to a new line as it is in the Figma design 

##### Your design
![Screenshot from 2022-05-26 18-39-47](https://user-images.githubusercontent.com/58771507/170544696-6c867ca6-982c-41de-a206-02e4fe574564.png)

##### Figma design 
![Screenshot from 2022-05-26 18-40-36](https://user-images.githubusercontent.com/58771507/170544808-2a651827-30b0-40b7-9657-3dd0af6305b5.png)